pkg/sql/logictest/testdata/logic_test/create_statements

```diff
@@ -1042,6 +1042,23 @@ CREATE TABLE crdb_internal.session_variables (
    value STRING NOT NULL,
    hidden BOOL NOT NULL
 )  {}  {}
+CREATE TABLE crdb_internal.statement_statistics (
```

Comment:
Looks good! Seems like the exact same schema as system.statement_statistics right? 

What's your rationale for making this a table vs a view?

Reply:
Using a table here is necessary because we are exposing cluster-wide in-memory data. This data is not accessible via SQL anywhere in the system and that is the reason why we need to create a virtual table to populate the data ourselves in Golang instead of SQL.